instrumentation/kamon-kafka/src/main/scala/kamon/instrumentation/kafka/client/KafkaInstrumentation.scala

```diff
@@ -194,12 +207,13 @@ object KafkaInstrumentation {
 
   object Keys {
     val Null = "NULL"
-    val ContextHeader = "kctx"
   }
 
-  case class Settings (
+  case class Settings(
     startTraceOnProducer: Boolean,
     continueTraceOnConsumer: Boolean,
-    useDelayedSpans: Boolean
+    useDelayedSpans: Boolean,
+    propagator: KafkaPropagator,
```

Comment:
Please remove this trailing comma so it compiles on earlier Scala versions

Reply:
👍  did the change